pom.xml

```diff
@@ -1989,6 +1989,16 @@
                 <artifactId>org.wso2.carbon.identity.organization.management.core.server.feature</artifactId>
                 <version>${identity.org.mgt.core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.auth.organization.login</groupId>
+                <artifactId>org.wso2.carbon.identity.auth.organization.login.server.feature</artifactId>
+                <version>${identity.organization.login.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.extension.identity.oauth2.grantType.organizationswitch</groupId>
```

Comment:
This group id has a bit different format. Is this used in any other oauth2-related extensions?

Reply:
Yes it is bit diferent. But same as here - https://github.com/wso2-extensions/identity-oauth2-grant-token-exchange/blob/main/pom.xml#L26